README.md

```diff
@@ -33,6 +33,12 @@ Active development occurs in this repository, which means some contracts in it m
 
 The build & test instructions below should work out of the box with Node ^14.18.0. (Please note that it needs Node 14 specifically, and will NOT work with Node 16 or higher. Minor version should be at least 18).
 
+To use the suggested Node version, run:
+
+```bash
+$ nvm use
+```
+
```

Comment:
This implies that nvm is used? I don't use nvm myself.

Reply:
yes; this is just a shortcut if you are lazy to install a version that works manually (the right versions are specified above these lines). Do you suggest rewording this?

Reply:
I suppose it could be generalized a bit, noting that using an incompatible Node version can cause issues, and that it's possible to install multiple Node versions using a manager such as nvm. If you use nvm, you can switch between them with `nvm use`. I think that's the most popular one, so reasonable to mention it like this.